README.md

```diff
@@ -1,18 +1,41 @@
-# Truffle-AI-Backend
+# truffle-ai-backend
 
-## Setup
+## Installation
 
-```bash
-git clone git@github.com:jst-seminar-rostlab-tum/truffle-ai-backend.git
-npm install
-npm run prepare
-npm run dev
+Install Docker from https://www.docker.com/ and install dependencies with `npm ci`.<br/>
+Before starting to develop check that you have setup your Environment variables in `.env.dev`! The script `env.sh` checks they are loaded during development.
```

Comment:
Add info on which env vars need to be set and where to find their values

Reply:
resolved